src/components/NavBar/NavBar.module.css

```diff
@@ -0,0 +1,35 @@
+.navBar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 16px;
+    background-color: #d4d4d4;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    width: 100%;
+    border-radius: 10px;
+    margin-right: 10px;
+    height: 70px;
+    box-sizing: border-box;
+  }
```

Comment:
In general, except for borders, it might be smarter to use em or rem rather than px.
I highly suggest you watch this before deciding: https://www.youtube.com/watch?v=XanhwddQ-PM&ab_channel=BocoupLLC